config/locales/en.yml

```diff
@@ -505,3 +526,5 @@ en:
             time_of_day:
               blank: Select a time of day
               inclusion: Select a time of day
+            patients:
+              blank: Cannot continue without selecting patients
```

Comment:
```suggestion
              blank: Select one or more patients to add to the session
```

Reply:
Fix from a mini snag session with Emma this morning. 🙏